front_end/handlers/DevelopmentLoginHandler.py

```diff
@@ -0,0 +1,37 @@
+from helper import *
+from tornado.web import *
+import traceback
+from content import *
+
+
+settings_dict = load_yaml_dict(read_file("/Settings.yaml"))
```

Comment:
I don't think we are using settings_dict. If not, please remove it. If we are, please declare it where it is being used.